### DISCUSSION TITLE
Sketchup Viewer

### DISCUSSION THREAD

**Discussion by oelamri** [elrumo/macOS_Big_Sur_icons_replacements#1324] [Icon request]
We need an icon for SketchUp Viewer: https://www.sketchup.com/products/sketchup-viewer

We already have icons for:
- Sketchup
- Sketchup Layout
- Sketchup Style Builder

We just need one for the Sketchup Viewer. It should be like the iPad icon: https://apps.apple.com/us/app/sketchup-viewer/id796352563

Whereas the Sketchup icon has a is blue logo on white background; the Sketchup Viewer icon should have a white logo on blue background (as in the Apple Store link above).

Thanks!

**Comment by oelamri** [comment]
@ChilliChompa Can I please get this one too?

**Comment by cake03** [comment]
![image](https://user-images.githubusercontent.com/80169337/141098088-22041551-678e-4f5d-9577-c9d5ca4e81a0.png)

**Reply by oelamri** [reply]
@C47F15H Thank you for these!

Can you please make one exactly like this one: https://apps.apple.com/us/app/sketchup-viewer/id796352563

It's better if it's flat and the background is blue, exactly like the iPad version. It matches their brand design style.

**Reply by elsiehupp** [reply]
@oelamri I just clicked on the link you shared, and it looks like Trimble updated the iPad icon to match the desktop one.
